Assets/Scripts/UI/InGame/Graphics/GeneralGraphics.cs

```diff
@@ -52,88 +53,50 @@ public Toggle CustomSettings
 			get { return customSettings; }
 			set { customSettings = value; }
 		}
+		
+		
+
+		public enum Textures
```

Comment:
Public enum should be a separate file, or make this private if it's only used within this class